WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/details/ThreatDetailsListItemsBuilder.kt

```diff
@@ -1,12 +1,44 @@
 package org.wordpress.android.ui.jetpack.scan.details
 
 import dagger.Reusable
+import org.wordpress.android.R
 import org.wordpress.android.fluxc.model.scan.threat.ThreatModel
+import org.wordpress.android.fluxc.model.scan.threat.ThreatModel.FileThreatModel
+import org.wordpress.android.fluxc.model.scan.threat.ThreatModel.FileThreatModel.ThreatContext
+import org.wordpress.android.fluxc.model.scan.threat.ThreatModel.FileThreatModel.ThreatContext.ContextLine
 import org.wordpress.android.ui.jetpack.common.JetpackListItemState
+import org.wordpress.android.ui.jetpack.scan.details.ThreatDetailsListItemState.ThreatContextLinesItemState
+import org.wordpress.android.ui.jetpack.scan.details.ThreatDetailsListItemState.ThreatContextLinesItemState.ThreatContextLineItemState
 import javax.inject.Inject
 
 @Reusable
 class ThreatDetailsListItemsBuilder @Inject constructor() {
     // TODO ashiagr to be implemented
```

Comment:
Tiny nitpick - Looks like you implemented this, do you want to keep the TODO any longer?

Reply:
This is not fully implemented in this PR. I've removed it in the parent PR #13703.